framework/src/modules/chain/chain.js

```diff
@@ -166,14 +158,68 @@ module.exports = class Chain {
 
 			await this._initModules();
 
+			// Prepare dependency
+			const processorDependency = {
+				blocksModule: this.blocks,
+				logger: this.logger,
+				constants: this.options.constants,
+				exceptions: this.options.exceptions,
+			};
+
+			// TODO: remove this once we have version 2 genesis block
+			this.processor.register(new BlockProcessorV0(processorDependency), {
+				matcher: ({ height }) => height >= 0 && height <= 0,
+			});
+
+			// TODO: Move this to core
```

Comment:
Could you create an issue to address this `TODO` and comment it in the code? I think it's good practice